rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -467,7 +467,12 @@ private J.EnumValue visitEnumField(FieldNode field) {
                     Delimiter delimiter = getDelimiter(null, cursor);
                     if (delimiter != null) {
                         cursor += delimiter.open.length();
-                        sourceBefore(delimiter.close);
+                        if (delimiter.close == "\"") {
```

Comment:
Did you mean the following? Or was this intentional? Rarely a problem in practice I think, but it looked out of place.
```suggestion
                        if ("\"".equals(delimiter.close)) {
```